requirements.txt

```diff
@@ -0,0 +1 @@
+gensim
```

Comment:
Can you set-up a virtual environment on your machine with [venv](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/) and run `pip freeze > requirements.txt` instead? I think this workflow will make setting up our python environment a lot easier and will make the conversion onto lambda easier as well since we know what packages and what versions we need for things!